castervoice/lib/ccr/core/text_manipulation.py

```diff
@@ -14,6 +14,8 @@
 from castervoice.lib.ccr.core.punctuation import text_punc_dict,  double_text_punc_dict
 from castervoice.lib.alphanumeric import caster_alphabet
 
+base_number_dict = {"zero": "0", "one": "1", "two": "2", "three": "3", "four": "4", "five": "5", "six": "6", "seven": "7", "eight": "8", "nine": "9" }
+number_dict = {"num "+k:base_number_dict[k] for k in base_number_dict}
```

Comment:
should be numb here.

Reply:
it seems to get misinterpreted as dictation less often when using numb for whatever reason